### PR TITLE
rails-blog-app: add views to the project

### PR DESCRIPTION
**Pull request by omarbabou**
- [ ] Implement the design from the sneak peek wireframes.
- [ ] Use methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- [ ] Add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).
- [ ] Fix linters errors

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      comment.author = user
+      comment.post = post
+      if comment.save
+        flash[:success] = 'Comment saved successfully'
+        redirect_to user_post_url(id: post.id)
+      else
+        flash[:error] = 'Error: Comment could not be saved'
+        redirect_to new_user_post_comment_url
+      end
+    end
```
Comment (elisha2kyakpo1) [inline app/controllers/comments_controller.rb]: - [x] I couldn't create a comment. Kindly fix this function, we are only told to create a current user function and not a current post. Users should be able to create comments and posts in the way they want, not only one post.

comm

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @omarbabou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ You have descriptive commit messages.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+	<meta charset="utf-8">
+	<title>See user's posts</title>
+</head>
+<body>
+  <main id ="container"> 
+  <%= render partial:'users/user', locals:{user: @user} %>
+   <% @posts.each do |post| %>
+  <%= render post%>
+   <section class="comment-section">
+    <%= render partial:'comments', as: 'comment', collection:post.recents_comments%>
```
Comment (omarsalem7) [inline app/views/posts/index.html.erb:13]: - [x] Great work in fixing the issue raised by the previous reviewer but after creating a comment, I got the following error, kindly fix it.
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @omarbabou , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ Descriptive commit message
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @omarbabou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Good communicate with reviewers ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**